### DISCUSSION TITLE
Highest Score

### DISCUSSION THREAD

**Discussion by kArenGos** [octokatherine/word-master#142] [General]
I'm wondering what the highest score is for consecutive wins on Word Master.
I got 172 and finally lost on the word easer; a word never spoken out loud by anyone ever.
My last high score was 132-135, I don't recall the actual score, but 172 is definitely phenomenal!
I'm super proud of my score and I am honored to have finally lost on such a ridiculous word.

I have screen shots of that game and my high score but do not know how to upload them.
I am not a techy and have no idea how to program or add content to git hub.
I'm just really curious about other high scores. 
Thanks
kAren

**Comment by kArenGos** [comment]
![Screenshot_20221202-142814_Chrome](https://user-images.githubusercontent.com/119709306/205421266-a24cf49e-f076-4001-b087-76e7855d7bac.jpg)
![Screenshot_20221202-142758_Chrome](https://user-images.githubusercontent.com/119709306/205421267-f7c7d18f-4ba7-4c1d-a9c0-87633d2a78c8.jpg)

**Reply by kArenGos** [reply]
I figured out how to add my screen shots.
Congratulations, prizes, and challenges are all welcome.

**Comment by Iggie99** [comment]
I got 206 on hard and missed out on sizer

The game frustrates the hell out of me though with all the ‘ed’ ‘er’ and plural words. These take all the skill out of the game and just turn it into a game of guess the consonant.

**Comment by Iggie99** [comment]
![IMG_3385](https://github.com/user-attachments/assets/1234475f-0652-4c01-9d05-11c948b7b4ee)
